docs/pages/application-access/cloud-apis/aws-console.mdx

```diff
@@ -269,44 +527,136 @@ AWS roles in these accounts.
 
-(!docs/pages/includes/aws-credentials.mdx service="the Application Service"!)
+If you deployed the Teleport Application Service on Kubernetes, it will have
+already started, and you can skip to [Step 4](#step-44-access-aws-resources).
 
-(!docs/pages/includes/start-teleport.mdx service="the Application Service"!)
+1. (!docs/pages/includes/aws-credentials.mdx service="the Application Service"!)
+1. (!docs/pages/includes/start-teleport.mdx service="the Application Service"!)
 
 <Admonition type="warning" title="non-standard AWS regions">
 For non-standard AWS regions such as AWS GovCloud (US) regions and AWS China
-regions, please set the corresponding region in the `AWS_REGION` environment
-variable or in the AWS credentials file so that the Application Service can use
-the correct STS endpoint.
+regions, set the corresponding region in the `AWS_REGION` environment variable
+or in the AWS credentials file so that the Application Service can use the
+correct STS endpoint.
 </Admonition>
 
-## Step 6/9. Connect to AWS console with assumed IAM role
+## Step 4/4. Access AWS resources
 
-Navigate to the Applications tab in your Teleport cluster's control panel and
-click on the Launch button for the AWS console application which will bring up
-an IAM role selector:
+Now that you have configured the Teleport Application Service to proxy requests
+to AWS, users can access AWS resources through Teleport.
+
+### Access the AWS console
+
+1. Visit the home page of the Teleport Web UI or click the **Resources** tab. If
+   the Teleport Application Service is proxying the AWS Management Console as
+   expected, the Web UI will display the name of the application you registered,
+   which this guide assumes is `awsconsole`. (Enter the name of the application
+   into the search box if there are too many resources to view all of them at
+   once.)
+
+1. Click the **Launch** button for the AWS Console application, then choose the
+   role you would like to assume when signing in to the AWS Console:
+
+   ![IAM role selector](../../../img/application-access/iam-role-selector.png)
 
-![IAM role selector](../../../img/application-access/iam-role-selector.png)
+1. Click on the role you want to assume and you will get redirected to the AWS
+   Management Console, signed in with the selected role. You should see your
+   Teleport user name as a federated login assigned to `ExampleReadOnlyRole` in
+   the top-right corner of the AWS Console:
 
-Click on the role you want to assume and you will get redirected to the AWS
-Management Console, signed in with the selected role.
+   ![Federated login](../../../img/application-access/federated-login@2x.png)
 
-In the console's top-right corner you should see that you're logged in through
-federated login and the name of your assumed IAM role:
+### Access the AWS CLI
 
-![Federated login](../../../img/application-access/federated-login@2x.png)
+1. Log into the previously configured AWS console app on your desktop:
```

Comment:
is the `previously` necessary? (b/c i didn't know what was previously configured)
```suggestion
1. Log into the configured AWS console app on your desktop:
```